components/camel-vertx/camel-vertx-http/src/main/java/org/apache/camel/component/vertx/http/VertxHttpEndpoint.java

```diff
@@ -60,18 +61,23 @@ public VertxHttpComponent getComponent() {
     @Override
     protected void doInit() throws Exception {
         String range = configuration.getOkStatusCodeRange();
+        parseStatusRange(range);
+    }
+
+    private void parseStatusRange(String range) {
         if (!range.contains(",")) {
-            // default is 200-299 so lets optimize for this
-            if (range.contains("-")) {
-                minOkRange = Integer.parseInt(StringHelper.before(range, "-"));
-                maxOkRange = Integer.parseInt(StringHelper.after(range, "-"));
-            } else {
+            if (!HttpUtil.parseStatusRange(range, this::setRanges)) {
                 minOkRange = Integer.parseInt(range);
                 maxOkRange = minOkRange;
             }
         }
     }
 
+    private void setRanges(int minOkRange, int maxOkRange) {
```

Comment:
Is this method in use ?

Reply:
Yes it is. It's referenced on line 69, by the `parseStatusRange`.